app/jobs/itms_job.rb

```diff
@@ -16,9 +16,9 @@ def self.downloader
     def self.href_exists?(href)
       downloader.url_exists?("#{API_ENDPOINT}#{href}")
     end
-    
-    def self.get_json_from_href(href)
-      response = get("#{API_ENDPOINT}#{href}")
+
+    def self.get_json_from_href(href, params = {})
+      response = get("#{API_ENDPOINT}#{URI(href).tap { |url| url.query = [url.query, params.reject { |_, v| v.blank? }.to_query].compact.join('&') }.to_s}")
```

Comment:
pozrel som este na krajne pripady ked `URI(href).query == nil` alebo `params == {}` a upravil som to takto:

```suggestion
      location = URI(href).tap { |u| u.query = URI.encode_www_form(URI.decode_www_form(u.query.to_s) + params.reject { |_, v| v.blank? }).presence }
      response = get("#{API_ENDPOINT}#{location}")
```